server/registry/internal/storage/errors.go

```diff
@@ -47,76 +48,82 @@ func AlreadyExists(err error) bool {
 
 // grpcErrorForDBError converts recognized database error codes to grpc error codes.
 func grpcErrorForDBError(ctx context.Context, err error) error {
+	// unwrap err, if wrapped
+	cause := errors.Cause(err)
```

Comment:
Had to look up [Cause](https://github.com/pkg/errors/blob/v0.9.1/errors.go#L275): "If the error does not implement Cause, the original error will be returned. If the error is nil, nil will be returned without further investigation."